package.json

```diff
@@ -1,33 +1,29 @@
 {
   "name": "appdev",
-  "version": "0.2.0",
-  "description": "AppDev Core JS Modules",
-  "main": "lib/Library.min.js",
+  "version": "0.4.0",
+  "description": "AppDev Core JavaScript Modules",
+  "main": "build/bundle.js",
   "scripts": {
-    "build": "node node_modules/webpack/bin/webpack.js --display-error-details -p",
-    "prepublish": "node node_modules/webpack/bin/webpack.js --display-error-details -p"
+    "build": "webpack -p",
```

Comment:
isn't it better to have `webpack` refer to a `node_modules` script, since running it in this way requires it to be globally installed?

Reply:
NPM will add the path automatically when doing `npm run XXX`, this is what we use in other JS backends